src/citrine/gemtables/variables.py

```diff
@@ -134,9 +139,10 @@ class AttributeByTemplate(Serializable['AttributeByTemplate'], Variable):
         a short human-readable name to use when referencing the variable
     headers: list[str]
         sequence of column headers
-    template: LinkByUID
+    template: Union[UUID, str, LinkByUID, AttributeTemplate]
         attribute template that identifies the attribute to assign to the variable
-    attribute_constraints: list[list[LinkByUID, Bounds]]
+    attribute_constraints: list[list[Union[UUID, str, LinkByUID, AttributeTemplate], Bounds]]
+        Optional
```

Comment:
Why is this on its own line?

Reply:
I'm open to cleaning this up, but the full string broke flake8 line length limits.  This seemed like the most logical solution.